soccer/gameplay/tactics/find_pass_point.py

```diff
@@ -0,0 +1,24 @@
+import robocup
+import constants
+from scipy.optimize import minimize
+
+## Function that optimizes cost function for passing
+# uses scipy minimize using the BFGS method
+def findPassPoint(heuristic) -> robocup.Point:
+    width = constants.Field.Width
+    length = constants.Field.Length
+    initial_guess = [0.00, 0.00]
```

Comment:
Why is this the initial guess? Let the user put their own initial guess

Reply:
Agreed.